zerver/lib/outgoing_webhook.py

```diff
@@ -236,10 +236,13 @@ def notify_bot_owner(
 ) -> None:
     message_url = get_message_url(event)
     bot_id = event["user_profile_id"]
-    bot_owner = get_user_profile_by_id(bot_id).bot_owner
+    bot = get_user_profile_by_id(bot_id)
+    bot_owner = bot.bot_owner
     assert bot_owner is not None
 
-    notification_message = f"[A message]({message_url}) triggered an outgoing webhook."
+    notification_message = (
+        f"[A message]({message_url}) to your @**{bot.full_name}** triggered an outgoing webhook."
```

Comment:
"To your bot ..." (I.e. add "bot " before the name) would be a good idea.

Also, I'd probably use the `@_` mention syntax -- we don't want this to trigger a notification to the bot under any circumstances.